src/cclib/io/moldenwriter.py

```diff
@@ -8,31 +8,62 @@
 """A writer for MOLDEN format files."""
 
 import os.path
+import math
 
 from . import filewriter
 from cclib.parser import utils
 
 
+def round_molden(num, p=6):
```

Comment:
Tests added [here](https://github.com/cclib/cclib/pull/391/files#diff-b77b9f83bdfaba6902ee0d0d99d8ac7dR79) to cover relevant cases of negative, positive integers and floats.